src/engine/source/conf/src/conf.cpp

```diff
@@ -83,8 +83,9 @@ Conf::Conf(std::shared_ptr<IApiLoader> apiLoader)
                          "/run/wazuh-server/engine.socket");
 
     // TZDB module
-    addUnit<std::string>(key::TZDB_PATH, "WAZUH_TZDB_PATH", "/var/lib/wazuh-server/engine/tzdb");
+    addUnit<std::string>(key::TZDB_PATH, "WAZUH_TZDB_PATH", "/var/ossec/engine/tzdb");
```

Comment:
It would be better to use the /var/ossec/queue/tzdb folder, since this is persisted by the cloud team.

Reply:
done